integration-tests/src/tests/network/routing.rs

```diff
@@ -1,57 +0,0 @@
-use crate::tests::network::runner::*;
-use near_network::time;
-
-#[test]
-fn account_propagation() -> anyhow::Result<()> {
```

Comment:
This test is redundant with the [do_not_block_announce_account_broadcast](https://github.com/near/nearcore/blob/f69ef1518a5cd499472b2c0418dd0bb49281c3ef/chain/network/src/peer_manager/tests/routing.rs#L1194) test.